mikelpardo/mobile-sales
# Get movile list from endpoint and add tests to the home view

**Description**

Get all moviles from given endpoint and add core and home view tests

**How to test**

1. Go to: http://localhost:3000/
2. Check in the console that it gets all movile list from the endpoint and display correctly in the view
